packages/code-du-travail-data/dataset/datafiller/fetch-requetes.js

```diff
@@ -29,10 +29,14 @@ const RECORDS_URL = `${DATAFILLER_URL}/kinto/v1/buckets/datasets/collections/req
 
 // fetch title from remote url
 const getPageTitle = async url => {
-  const text = await fetch(url).then(r => r.text());
-  const matches = text.match(/<title>([^<]+)<\/title>/i);
-  if (matches) {
-    return decodeHTML(matches[1]);
+  try {
+    const text = await fetch(url).then(r => r.text());
+    const matches = text.match(/<title>([^<]+)<\/title>/i);
+    if (matches) {
+      return decodeHTML(matches[1]);
+    }
+  } catch (e) {
+    //console.log(e);
```

Comment:
console.error() will not pollute output thus, allow to know wich url fails
```suggestion
    console.error(`fail to retrieve tittle on ${url}`);
    console.error(e);
```